### PR TITLE
Tickets/dm 45110

### PR DESCRIPTION
**Pull request by teweitsai**
* Improve the **README.md**.
* Update the ``MainWindow._callback_disconnect()`` to check the current commander.
* Add the ``SignalClosedLoopControlMode`` and update ``Model._process_event()`` to send the signal.
* Add the ``TabOverview._set_signal_closed_loop_control_mode()``. 
* Add the ``TabIlcStatus._is_closed_loop_control_mode_in_idle()``.
* Update the **TabDiagnostics** to support the control of closed-loop control mode.
* Add the ``TabSettings._callback_time_out()`` to write the external elevation angle continuously.
* Update the user manual and UML.

### CODE REVIEW

File: python/lsst/ts/m2gui/main_window.py [hunk 1]
```diff
@@ -434,7 +434,7 @@ def _create_dialog_exit(self) -> QMessageBoxAsync:
 
         Returns
         -------
-        `QMessageBoxAsync`
+        dialog : `QMessageBoxAsync`
```
Comment (sebastian-aranda) [inline python/lsst/ts/m2gui/main_window.py:437]: You removed the return names of a few functions on `tab_diagnostics.py` (I assume, based on [this](https://numpydoc.readthedocs.io/en/latest/format.html#returns)) so you might want to keep the same standard here?

File: python/lsst/ts/m2gui/main_window.py [hunk 2]
```diff
@@ -507,6 +517,33 @@ async def _callback_disconnect(self) -> None:
         action_disconnect.setEnabled(True)
         action_exit.setEnabled(True)
 
+    def _create_dialog_disconnect(self) -> QMessageBoxAsync:
+        """Create the disconnect dialog.
+
+        Returns
+        -------
+        dialog : `QMessageBoxAsync`
```
Comment (sebastian-aranda) [inline python/lsst/ts/m2gui/main_window.py:525]: Similar to a comment above, you might want to remove the return name?

**Review by sebastian-aranda — APPROVED** [review]
This looks good to me :+1:. I left a couple comments @teweitsai , but the PR is approved anyways.

File: python/lsst/ts/m2gui/main_window.py (hunk 1 above)
Comment (teweitsai) [reply python/lsst/ts/m2gui/main_window.py:437]: This function here does do the return:

```
return dialog
```

The removed parts in `tab_diagnostics.py` do not.

Thanks!

File: python/lsst/ts/m2gui/main_window.py (hunk 2 above)
Comment (teweitsai) [reply python/lsst/ts/m2gui/main_window.py:525]: Same comment above. Thanks!

File: doc/uml/controltab/class_tab_diagnostics.mmd [hunk 3]
```diff
@@ -4,9 +4,11 @@ namespace m2gui {
   class UtilityMonitor
   class SignalUtility
   class SignalDetailedForce
+  class SignalClosedLoopControlMode
 }
 
 Model *-- UtilityMonitor
+Model *-- SignalClosedLoopControlMode : emit()
```
[the remark this answers is not in the document]
Comment (teweitsai) [reply doc/uml/controltab/class_tab_diagnostics.mmd:11]: Improved in the commit: [e95226b](https://github.com/lsst-ts/ts_m2gui/pull/76/commits/e95226b2f751504e7eb3ef77151a43fae967f1d1). Thanks!

File: doc/user-guide/user-guide.rst [hunk 4]
```diff
@@ -377,6 +377,10 @@ It is noted that only when the D2 ("interlock enable" in digital output) and D31
 This implies that you could manually toggle on/off the D0 and D1 to check the interlock signal exists or not.
 If you do so, you always need to turn on the communication power first followed by the motor power, and turn off the motor power first followed by the communication power.
 
+You can use the **Update Control Mode** button to change the current mode (see **Mode** combo selection) in closed-loop control mode.
+You might need to use this function in caution because this might affect the use of hardware.
```
[the remark this answers is not in the document]
Comment (teweitsai) [reply doc/user-guide/user-guide.rst]: Improved in the commit: [e95226b](https://github.com/lsst-ts/ts_m2gui/pull/76/commits/e95226b2f751504e7eb3ef77151a43fae967f1d1). Thanks!

**Pull request merged**